packages/react-icons-mdl2/src/utils/createSvgIcon.ts

```diff
@@ -3,10 +3,12 @@ import { css, getNativeProps, htmlElementProperties } from '@fluentui/utilities'
 import * as classes from './SvgIcon.scss';
 import { ISvgIconProps } from './SvgIcon.types';
 import { SvgIconCreateFnParams } from './types';
+import { useIcon } from '@fluentui/react-icon-provider';
 
 const createSvgIcon = <TProps = {}>({ svg, displayName }: SvgIconCreateFnParams<TProps>) => {
   const Component: React.FC<React.HTMLAttributes<HTMLSpanElement> & TProps & ISvgIconProps> = props => {
     const { className, style = {} } = props;
+    const icons = useIcon();
```

Comment:
`icons` also contains `fontFace` and `styles` which you also need to handle

Reply:
For right now, I'm focusing on implementing the provider just to provide icon overrides, and will be adding this as necessary

Reply:
sounds good, pls add comment `// TODO: handle `fontFaces` and `styles` here to track